cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1723,6 +1723,46 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Google Workspace
 
 
+[Google Workspace - Google Drive objects accessed by External User]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 58 * * * *
+description =  The alert will generate if google drive or google docs files accessed by external user. \
+NOTE: cs_email_user_domain macro must be configured with the domain name in order to use this alert. \
+Data Collection - Google drive activity reports data from Splunk Add-on for Google Workspace (https://splunkbase.splunk.com/app/5556) 
+dispatch.earliest_time = -6h@h
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_gws` sourcetype="gws:reports:drive" _index_earliest=-61m@m _index_latest=-1m@m \
+| rex field=owner "[^@]+@(?<src_domain>[^@]+)" \
+| rex field=email "[^@]+@(?<dest_domain>[^@]+)" \
+| where isnotnull(src_domain) AND isnotnull(dest_domain) \
+| search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain` \
+| `cs_user_privilege_mapping(email)` \
```

Comment:
| `cs_user_privilege_mapping(owner)` \

we have to privilege mapping with internal users.